packages/model-viewer/src/test/index.ts

```diff
@@ -51,6 +51,7 @@ import './features/scene-graph/model-spec.js';
 import './features/ar-spec.js';
 import './features/scene-graph/texture-spec.js';
 import './features/scene-graph/material-spec.js';
+import './features/scene-graph/texture-info-spec.js';
```

Comment:
Did you forget to commit `texture-info-spec`?

Reply:
oops